Makefile

```diff
@@ -141,6 +155,12 @@ patch_disable_keyboard:
 	 $(call patch_disable,$$j); \
 	done
 
+patch_enable_fn_ctrl_swap:
+	$(call patch_enable,008_fn_ctrl_swap.patch)
+
+patch_disable_fn_ctrl_swap:
+	$(call patch_disable,008_fn_ctrl_swap.patch)
+
```

Comment:
True. One the other hand, three patches are not that much either. If more get added then it should probably be optimized.